pyproject.toml

```diff
@@ -25,7 +27,7 @@ markers = "fuzzing: Run Hypothesis fuzz test suite"
 line_length = 100
 force_grid_wrap = 0
 include_trailing_comma = true
-known_third_party = ["IPython", "click", "dataclassy", "eth_abi", "eth_account", "eth_typing", "eth_utils", "geth", "github", "hexbytes", "hypothesis", "hypothesis_jsonschema", "importlib_metadata", "pluggy", "pytest", "requests", "setuptools", "web3", "yaml"]
+known_third_party = ["IPython", "_pytest", "click", "dataclassy", "eth_abi", "eth_account", "eth_typing", "eth_utils", "geth", "github", "hexbytes", "hypothesis", "hypothesis_jsonschema", "importlib_metadata", "pluggy", "pytest", "requests", "setuptools", "web3", "yaml"]
```

Comment:
why underscore?

Reply:
I am not sure, this got added automatically

Reply:
It was because we were importing the `Config` class from `_pytest` but that was related to the xdist check (which I just removed).